### ISSUE TITLE
Statfactory?

### ISSUE THREAD

**Issue by Uziel9999** [Tsunami-ide/public#16]
I get the error "Assembly reference ... was not found or is invalid"  What have I done wrong?
# r @"C:\Program Files (x86)\Statfactory\FCell 1.0\FCell.ManagedXll.dll"
# r @"C:\Program Files (x86)\Statfactory\FCell 1.0\FCell.ManagedXll.Rtd.dll"
# r @"C:\Program Files (x86)\Statfactory\FCell 1.0\log4net.dll"

Thank you in advance.

**Comment by moloneymb** [comment]
Have you installed FCELL?

**Comment by Uziel9999** [comment]
No, I just downloaded Tsunami from your site.  I cannot find any installation instructions nor do I find anything indicating the need for FCell.  

Does this mean that we must purchase FCell in order to use Tsunami?

**Comment by moloneymb** [comment]
No you only need FCell in order to load dlls into Excel. Tsunami and FCell work independently. 

You can communicate with Excel from Tsunami using the COM dlls.

The plug-in to install Tsunami into Excel is a part of Tsunami and is done on running the MSI. If you don't see Tsunami in the Ribbon then it didn't work and your out of luck for the embedding. Excel is pretty finicky and a decent number of times the plug-in wont install but we've unable to reproduce on any of our test machines to fix it.
